### ISSUE TITLE
Left click menu broken in 3.x

### ISSUE THREAD

**Issue by bbrala** [swisnl/jQuery-contextMenu#632]
See https://swisnl.github.io/jQuery-contextMenu/3.x/demo/trigger-left-click.html

**Comment by LordRampantHump** [comment]
thats true, however when adding  trigger: 'left', to the default demo it does work
